vignettes/estimate_infections.Rmd

```diff
@@ -33,31 +33,35 @@ These infections are then mapped to observations via discrete convolutions with
 
 ### Initialisation
 
-The model is initialised before the first observed data point by assuming constant exponential growth for the mean of modelled delays from infection to case report (called `seeding_time` $t_\mathrm{seed}$ in the model):
+The model is initialised before the first initerved data point by assuming constant exponential growth for the mean of modelled delays from infection to case report (called `seeding_time` $t_\mathrm{seed}$ in the model):
 
 \begin{align}
-  I_0  &\sim \mathrm{LogNormal}(I_\mathrm{obs}, \sqrt(I_\mathrm{obs})) \\
-  r &\sim \mathrm{Normal}(r_\mathrm{obs}, 0.2)\\
-  I_{0 < t \leq t_\mathrm{seed}} &= I_0 \exp  \left(r t \right)
+  I_0  &\sim \mathrm{LogNormal}(I_\mathrm{init}, \sqrt{I_\mathrm{init}}) \\
+  r &\sim r_\mathrm{init} + (I_\mathrm{init} - I_0) \\
```

Comment:
```suggestion
  r &\sim r_\mathrm{init} + (I_\mathrm{init} - I_0) \\
```

looking at this reminds me to ask did you look at this both normalised by the standard deviation and not?

Reply:
I initially had it divided by the seeding time but that was fairly poorly motivated. I assumed that we'd replace this by the R->r solution anyway so didn't dwell on it too much but if you can think of a more appropriate scaling factor here then this would probably be a good thing to include.

Reply:
my suggestion was the standard deviation so the scaling is the same regardless of the magnitude of the initial infections. I don't think we expect it to scale with the count magnitude do we?

Reply:
Perhaps the best thing is to just discard the approach and go straight with https://github.com/epiforecasts/EpiNow2/issues/920#issue-2780030137 rather than trying to come up with something good here.